src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.Api/Controllers/ProcessController.cs

```diff
@@ -433,5 +456,14 @@ private void NotifyAppAboutEvents(Instance instance, List<InstanceEvent> events)
                 }
             }          
         }
+
+        private string GetActionType(string currentTask)
+        {
+            if (currentTask.Equals("data")) {
```

Comment:
Why do we have write here? User already must have write scope to be allowed to put to the process controller. It does not make sense to check for data and return write. Action should be next and not write!!!!!

Reply:
@dskogan  If we gonna have next we would need to have a separate rule for this in the xacml file saying the user with role x can do next operation on a given task. We discussed this in #2521 and i thought you agreed on that.